omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/visit/ParserEncounterIntoSimpleObjects.java

```diff
@@ -125,6 +128,12 @@ public int compare(SimpleObject o1, SimpleObject o2) {
 	}
 	
 	private SimpleObject parseObs(Obs obs, Locale locale) {
```

Comment:
Concept concept = obs.getConcept();?

Reply:
If i use `Concept concept = obs.getConcept();` instead of  `Concept concept = HibernateUtil.getRealObjectFromProxy(obs.getConcept());` the concept will not be a conceptNumeric.

Reply:
Good idea.